### ISSUE TITLE
Invalid fields for entrytype 'set' with `--validate-datamodel`

### ISSUE THREAD

**Issue by brkalmar** [plk/biber#264]
**biber version**: 2.12 (default data model)
**system**: debian linux 4.19-0-4-amd64

Using `@set` with subentries in its `entryset` field leads to warnings about invalid fields from the subentries, and about a missing mandatory field `crossref`.

Example:
```latex
\documentclass{article}

\begin{filecontents}{\jobname.bib}
@set{a,
  entryset = {a1,a2},
}
@book{a1,
  author = {Author, A.},
  date = {2000},
  title = {A Book},
}
@book{a2,
  author = {Buthor, B.},
  date = {2001},
  title = {B Book},
}
\end{filecontents}

\usepackage{biblatex}
\addbibresource{\jobname.bib}

\begin{document}

\textcite{a}.

\printbibliography{}

\end{document}
```

Gives the following warnings:
```
$ biber --validate-datamodel test
...
WARN - Datamodel: Entry 'a' (test.bib): Invalid field 'author' for entrytype 'set'
WARN - Datamodel: Entry 'a' (test.bib): Invalid field 'title' for entrytype 'set'
WARN - Datamodel: Entry 'a' (test.bib): Missing mandatory field 'crossref'
...
```

These warnings are not debilitating on their own, as they could be ignored, and the document output seems to be as expected.  However, I like to use `biber` with `--validate-datamodel --dieondatamodel` in my build process, and this issue completely negates that possibility.

None of these warnings make sense considering the usage of `@set` given in the documentation (unless I've severely misunderstood something).  Neither have I found any further information searching anywhere.

**Comment by plk** [comment]
Agreed, this makes no sense. Looks like some legacy default datamodel constraints which I've removed in biber 2.13 DEV and biblatex 3.13 DEV which you can find updated on SourceForge shortly.

**Comment by plk** [comment]
Fixed in latest 2.13 release.
